fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -820,13 +820,15 @@ public Status resetIdsForRestore(GlobalStateMgr globalStateMgr, Database db, int
         setId(globalStateMgr.getNextId());
 
         // reset all 'indexIdToXXX' map
+        Map<Long, MaterializedIndexMeta> origIndexIdToMeta = Maps.newHashMap(indexIdToMeta);
+        indexIdToMeta.clear();
```

Comment:
It‘s a possible modifications, but I still prefer the original code, because the member variable may use various map types such as hash map, tree map, concurrent map etc..., the original code does not need to care about the real type of the member variable, even if the member variable changes it's real type.

Reply:
ok